bridget/eddy/footprint.py

```diff
@@ -15,13 +15,22 @@
 
 """
 
-def footprint():
+def footprint(height = 2, t_air, a, p, wndspeed, cov_uw, cov_vw, var_v, method = "Kormann_Meixner"):
     """
-    Top-level footprint function. 
+    Top-level footprint function.
 
-    If more than one footprint method is implemented, these should go 
-    into functions in this module. This function then has a **string** 
+    If more than one footprint method is implemented, these should go
+    into functions in this module. This function then has a **string**
     parameter to switch the method.
-    
+
+    Parameters:
+        - height = measurement height (z) [m]
+        - wndspeed = horizontal wind component for the direction in which the sonic is oriented (u) [m/s]
+        - cov_uw = Covariance between the wind components u and w
+        - cov_vw = Covariance between the wind components v and w
+        - var_v = variance of v (horizontal wind component for the direction rectangular to the orientation of the sonic
+        - t_air = air temperature [°C]
+        - a = absolute humidity [g/m3]
+        - p = air pressure [hPa]
```

Comment:
It's just personal flavor, but I prefer [numpy-style docsstrings](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard) but that is not a high priority.